book/src/template_syntax.md

```diff
@@ -914,9 +914,54 @@ Then if you don't pass a value for this argument, its default value will be used
 
 ```jinja
 {# We only specify `arg1`, so `arg2` will be "something" #}
-{% call heading(1) %}
+{% call heading(1) %} {% endcall %}
 {# We specify both `arg1` and `arg2` so no default value is used #}
-{% call heading(1, 2) %}
+{% call heading(1, 2) %} {% endcall %}
+```
+
+### Call
+
+you can specify a block for a call to be passed down as used as apart of a macro. For this case 
+a block is denoted by `{% call(args) macro() %}`, ending with `{% endcall %}`. the callee is denoted by
+`{{caller()}}`.
+
+```jinja
+{% macro render_dialog(title, class='dialog') -%}
+    <div class="{{ class }}">
+        <h2>{{ title }}</h2>
+        <div class="contents">
+            {{ caller() }}
+        </div>
+    </div>
+{%- endmacro %}
+
+{% call() render_dialog('Hello World') %}
+    This is a simple dialog rendered by using a macro and
+    a call block.
+{% endcall %}
+
+```
+
+here is an example with a call block using arguments:
+
+```jinja
+{% macro dump_users(users) -%}
+    <ul>
+    {%- for user in users %}
+        <li><p>{{ user.username }}</p>{{ caller(user) }}</li>
+    {%- endfor %}
+    </ul>
+{%- endmacro %}
+
+{% call(user) dump_users(list_of_user) %}
+    <dl>
+        <dt>Realname</dt>
+        <dd>{{ user.realname }}</dd>
+        <dt>Description</dt>
+        <dd>{{ user.description }}</dd>
+    </dl>
+{% endcall %}
+
```

Comment:
```suggestion
{% endcall %}
```

Reply:
Hum, weird suggestion... Well, please remove the empty line at the end of the block.